### PR TITLE
Crypto App

### PR DESCRIPTION
**Pull request by wadaudzai**
Dear Reviewer,

This project was built to display a website about cryptocurrencies. On this website you can view 100 cryptos their name, rank, and price on the home page the homepage can be filtered and you can search for your desired crypto among them. You can click on one of the cryptos and it will lead you to the details page of the crypto there you can see more information regarding the crypto that you have selected. All these data are fetched from the API and are shown on the screen here.

The link to the Loom Video: https://www.loom.com/share/bf246427ec944c3ca3bd23b1f364e740
I have worked on this project to the best of my knowledge. Please check it if you got time. Thanks for your time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/wadaudzai/react-capstone/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```
Comment (mirouhml) [inline README.md:49]: - [x] Please know that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @wadaudzai, 

Good job on finishing your capstone project, I loved it!

There are just a few things you forgot to include in the documentation that need to be there, I'll mention them below!

## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️


## Required changes
_Check the comments under the review._

**Please know that as per the requirements of the project, you need to record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:**

- **A description of the project.**
- **A demo of the project features.**
- **You should also highlight some interesting piece of code or something you built that you are very proud of.**

**For recording the video you can use tools like [Loom](https://www.loom.com/) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture, and then include it in your description in the pull request!**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (wadaudzai) [reply README.md:49]: Changes are made according to what you instructed. Thank you !

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @wadaudzai, 

Great work on making the changes 👏 

It would be better to include links to the designer and the website, it adds more credibility and value to your acknowledgment, this is just a suggestion from me. 

**The required change that I am requesting is that you didn't include a video, I already mentioned in my previous message that you should include a video as it is part of the requirements, It is a video with a maximum length of 5 minutes. The content of the video should include:**

- **A description of the project.**
- **A demo of the project features.**
- **You should also highlight some interesting piece of code or something you built that you are very proud of.**

**For recording the video you can use tools like [Loom](https://www.loom.com/) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture, also, please include everything mentioned in the requirements, and then kindly include it in your description in the pull request!**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by wadaudzai** [comment]
Hope its okay this time.

**Comment by wadaudzai** [comment]
Dear Reviewer you can see the link to the Loom video in the first comment of the pull request. Thank you!

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @wadaudzai, 

Thanks for recording the video, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**